GitUI/UserControls/RevisionGrid/ArtificialCommitChangeCount.cs

```diff
@@ -8,13 +8,18 @@ namespace GitUI
 {
     public sealed class ArtificialCommitChangeCount
     {
+        /// <summary>
+        /// If the count is valid or if it is updating.
+        /// </summary>
+        public bool Valid { get; private set; } = false;
```

Comment:
This is being set when we call `Update`:
```suggestion
        public bool IsUpdating { get; private set; } = false;
```

Reply:
Not only, also when commitCount is not used.
I can change the name and describe this, so it is easier to follow in the default configuration.